src/chromium/crInput.ts

```diff
@@ -33,18 +35,36 @@ function toModifiersMask(modifiers: Set<types.KeyboardModifier>): number {
 }
 
 export class RawKeyboardImpl implements input.RawKeyboard {
-  private _client: CRSession;
+  constructor(
+    private _client: CRSession,
+    private _isMac: boolean,
+  ) { }
 
-  constructor(client: CRSession) {
-    this._client = client;
+  _commandsForCode(code: string, modifiers: Set<types.KeyboardModifier>) {
+    if (!this._isMac)
+      return [];
+    const parts = [];
+    for (const modifier of (['Shift', 'Control', 'Alt', 'Meta']) as types.KeyboardModifier[]) {
+      if (modifiers.has(modifier))
+        parts.push(modifier);
+    }
+    parts.push(code);
+    const shortcut = parts.join('+');
+    let commands = (this._isMac && macEditingCommands[shortcut]) || [];
```

Comment:
this can be dropped due to fast-return on line 44

Reply:
```suggestion
    let commands = macEditingCommands[shortcut] || [];
```